src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -182,6 +183,9 @@ static int add_cryptsetup(const char *id, const char *what, bool rw, bool requir
         }
 
         return 0;
+#else
+        return log_debug_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "Encrypted partition, please set -Dlibcryptsetup=true to support it");
```

Comment:
hmm, actually, this should be upgraded to `log_error_errno()`, i.e. it's a fatal error and this function otherwise log at error level for fatal things.

"… to support it" → "… to enable support."